src/webhook-proxy.ts

```diff
@@ -0,0 +1,23 @@
+import * as Logger from 'bunyan'
+
+export const createWebhookProxy = (opts: WebhookProxyOptions) => {
+  try {
+    const SmeeClient = require('smee-client')
+
+    const smee = new SmeeClient({
+      source: opts.url,
+      target: `http://localhost:${opts.port}${opts.path}`,
+      logger: opts.logger
+    })
+    return smee.start()
```

Comment:
I'm not seeing type declarations for this. Where do they come from?

Reply:
I believe `tsc` did not complain about missing definitions probably because the return value of `createWebhookProxy` is never used (see https://github.com/probot/probot/blob/typescript-init/src/index.ts#L89-L96 ).